aii-pxelinux/src/test/perl/unconfigure.t

```diff
@@ -0,0 +1,68 @@
+use strict;
+use warnings;
+use Test::More;
+use Test::Quattor qw(pxelinux_base_config);
+use NCM::Component::PXELINUX::constants qw(:pxe_variants :pxe_constants);
+use NCM::Component::pxelinux;
+use CAF::FileReader;
+use CAF::Object;
+use Sys::Hostname;
+use File::Basename qw(dirname);
+use Readonly;
+use testpath;
+
+
+=pod
+
+=head1 SYNOPSIS
+
+Tests for the C<Unconfigure> method.
+
+=cut
+
+Readonly my $NBPDIR_GRUB2_VALUE => '/grub/config_files';
+Readonly my $NBPDIR_PXELINUX_VALUE => '/pxelink/config_files';
+Readonly my $FILE_INITIAL_CONTENTS => 'File has been configured';
+Readonly my $SYMLINK_INITIAL_CONTENTS => 'Symlink has been defined';
+
+$CAF::Object::NoAction = 1;
```

Comment:
ah no, those are your messages. but CAF::Path should report it's own if reporter is properly setup. but i guess this is not the case here, so that's why all the `this_app` is used.